app/src/main/java/org/p2p/wallet/root/DecorSystemBarsDelegate.kt

```diff
@@ -0,0 +1,68 @@
+package org.p2p.wallet.root
+
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.view.WindowCompat
+import androidx.core.view.WindowInsetsControllerCompat
+import androidx.lifecycle.DefaultLifecycleObserver
+import androidx.lifecycle.LifecycleOwner
+import android.graphics.Color
+import android.os.Build
+
+enum class SystemIconsStyle {
+    BLACK, WHITE
+}
+
+class DecorSystemBarsDelegate constructor(
```

Comment:
this class is responsible for system icons colors. For now we have only one theme, style: `SystemIconsStyle.BLACK` and sometimes need to set `SystemIconsStyle.WHITE`
But in future we can add dark theme and default `navigationBarStyle` `statusBarStyle` may change